src/x/mmap/mmap_linux.go

```diff
@@ -109,3 +109,15 @@ func Munmap(b []byte) error {
 
 	return nil
 }
+
+// Free mmapped memory. `MADV_DONTNEED` informs the kernel to free the mmapped pages right away instead
+// of waiting for memory pressure.
+// NB(bodu): DO NOT FREE anonymously mapped memory or else it will null all of the underlying bytes as the
+// memory is not file backed.
+func Free(b []byte) error {
```

Comment:
How do you feel about calling this `DontNeed` or something? Free makes me think looking through the code that the mmap should be unusable after that but thats not the case here

Reply:
Makes sense.  Perhaps `MadviseDontNeed(..)` to match the naming of `Munmap(...)`?